### DISCUSSION TITLE
Text formatting for axis labels

### DISCUSSION THREAD

**Discussion by bmccorm2** [observablehq/plot#1755] [Q&A] [answered]
Sorry i can't seem to find this anywhere.  Anyone know how i format an axis label (i.e., fontSize: 16 - and just the label not the actual axis)

Thanks!

**Comment by Fil** [comment, marked as answer]
A way to do this is to create two axes with the [axis mark](https://observablehq.com/plot/marks/axis#axis-options); one that only has the label (and the fontSize option), the other with only the ticks.

**Comment by bmccorm2** [comment]
Thanks that works to an extent.  It would be nice to have some more control of the labeling.  I'm not trying to go overboard but when i make the label the size i want it will overwrite the ticks.

<img width="435" alt="image" src="https://github.com/observablehq/plot/assets/11781610/c901a084-1025-4851-a25f-00385c1fdd85">

**Reply by Fil** [reply]
~~Have you tried using **dy** to move it vertically? (Sorry for the belated reply.)~~

**Reply by bmccorm2** [reply]
Ironically that dy offset is working for everything except the label.  (i.e., i can offset the ticks but not the actual label)

**Reply by Fil** [reply]
Right, it is labelOffset in that case. See https://observablehq.com/@recifs/plot-large-axis-label--support (and https://github.com/observablehq/plot/issues/1414)

**Comment by bmccorm2** [comment]
That's good thanks!  I used it in conjunction with marginBottom and works great.  Thank you so much!
